dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java

```diff
@@ -102,18 +102,18 @@ public String getToken() {
   public String getApiVersion() {
     return finalConfig.getString(URI_API_VERSION, "v3");
   }
-  
+
   @SuppressWarnings("unchecked")
   public List<String> getFileSources() {
-	Object property = finalConfig.getProperty(FILE_SOURCE);
-	if (property instanceof String) {
-	  List<String> result = new ArrayList<>();
-	  result.add((String) property);
-	  return result;
-	} else if (property instanceof List) {
-	  return (List<String>) property;
-	}
-	return new ArrayList<>();
+    Object property = finalConfig.getProperty(FILE_SOURCE);
+    if (property instanceof String) {
+      List<String> result = new ArrayList<>();
+      result.add((String) property);
+      return result;
+    } else if (property instanceof List) {
+      return (List<String>) property;
+    }
+    return new ArrayList<>();
```

Comment:
`return Collections.EMPTY_LIST` is better

Reply:
done